integration/apps/apps.py

```diff
@@ -69,26 +69,34 @@ def get_exec_args(self):
         ''' Command line arguments to the application. '''
         return self._exec_args
 
+    def get_custom_geopm_args(self):
+        ''' Additional geopm arguments required for the app, such as --geopm-ompt-disable.'''
+        return []
+
     def cleanup(self):
         ''' Any steps to be run after running one iteration of the application. '''
         return ''
 
     def parse_fom(self, log_path):
         return None
 
-    def make_bash(self, output_dir):
+    def make_bash(self, output_dir, run_id):
+        # setup has side effects; call before get_exec_args
+        setup = self.setup(run_id)
         app_params = self.get_exec_args()
         if type(app_params) is list:
             app_params = ' '.join(self.get_exec_args())
 
         script = '''#!/bin/bash\n'''
+        # TODO: cd interferes with AgentConf ability to correctly write the agent policy file
```

Comment:
check if this is still true; problem is with controller read, not agent conf write

Reply:
Is this still a problem?